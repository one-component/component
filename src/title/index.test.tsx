import React from 'react'
import renderer from 'react-test-renderer'
import { Title } from './index'

test('Title render', () => {
  const t = 'Foo render'
  const c = renderer.create(<Title title={t} />)
  const tree = c.toJSON()
  expect(tree).toMatchSnapshot()
})
