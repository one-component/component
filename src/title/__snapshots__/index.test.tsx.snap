// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Foo render 1`] = `
<h1>
  Foo render
</h1>
`;
